### PR TITLE
chore(package): update dependencies

### LINKED ISSUE #2387: An in-range update of prettier is breaking the build 🚨
## The devDependency [prettier](https://github.com/prettier/prettier) was updated from `1.17.1` to `1.18.0`.

🚨 [View failing branch](https://github.com/w3c/respec/compare/develop...w3c:greenkeeper%2Fprettier-1.18.0).

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

prettier is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.

<details>
<summary>Status Details</summary>

- ❌ **Travis CI - Branch:** <a href='https://travis-ci.com/w3c/respec/builds/114669606'> The build</a> **failed**. 
</details>

---

<details>
<summary>Release Notes for 1.18.0</summary>

<p><g-emoji class="g-emoji" alias="link" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f517.png">🔗</g-emoji> <a href="https://prettier.io/blog/2019/06/06/1.18.0.html" rel="nofollow">Release Notes</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by saschanaz**
I love this change :tada:

Closes #2387

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -148,9 +148,7 @@ function createIdlAnchor(escaped, data, parent) {
   if (showWarnings) {
     const styledName = data.type === "operation" ? `${name}()` : name;
     const ofParent = parentName ? ` \`${parentName}\`'s` : "";
-    const msg = `Missing \`<dfn>\` for${ofParent} \`${styledName}\` ${
-      data.type
-    }. [More info](https://github.com/w3c/respec/wiki/WebIDL-thing-is-not-defined).`;
+    const msg = `Missing \`<dfn>\` for${ofParent} \`${styledName}\` ${data.type}. [More info](https://github.com/w3c/respec/wiki/WebIDL-thing-is-not-defined).`;
```
Comment (sidvishnoi) [inline src/core/webidl.js]: Slightly concerned about this one

File: src/w3c/headers.js [hunk 2]
```diff
@@ -430,11 +426,7 @@ export function run(conf) {
     });
   if (conf.bugTracker) {
     if (conf.bugTracker.new && conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>${
-        conf.l10n.file_a_bug
-      }</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${
-        conf.l10n.open_bugs
-      }</a>${conf.l10n.close_parens}`;
+      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
```
Comment (sidvishnoi) [inline src/w3c/headers.js]: Slightly more concerned with this one 😄 
But I guess it's fine. Every editor supports word wrap.

File: package-lock.json [hunk 3]
```diff
@@ -3364,7 +3364,7 @@
         },
         "string_decoder": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (sidvishnoi) [inline package-lock.json]: Why they went from `https` to `http` 😕  ?

Comment (saschanaz) [reply package-lock.json]: It's a painful known bug from npm: https://npm.community/t/package-lock-json-keeps-changing-between-platforms-and-runs/1129/8

But there's a suggestion so I might try it.

Comment (saschanaz) [reply package-lock.json]: Just tried it and it even replaces existing `http:`. Yay!

File: src/w3c/headers.js (hunk 2 above)
Comment (saschanaz) [reply src/w3c/headers.js]: @marcoscaceres Why do we even have `l10n.open_parens` when it's always `(`?

**Pull request merged**